docs/concepts/PrivatePayment.md

```diff
@@ -0,0 +1,12 @@
+# Private Payment
+
+Manta is all about bringing privacy to Web 3.  Supporting privacy in fungible token transfer is essential. The first and most powerful primitive we are building is _Private Payment_, more specifcally, *MantaPay*, a _multi-asset decentralized anonymous payment protocol_. 
+
+The fundamental difference between Manta and other public blockchains is that the fungbile token transfer (a.k.a. payment) is shielded by zero-knowledge proofs and thus to be private. We illustrate the difference below:
+
+![private payment](./resources/private-payment.png)
+
+## Bring Your Own Token Privacy
+
+One difference between Manta and ZCash is that Manta supports BYOT (Bring Your Own Token) privacy. Manta boostraps from Polakdot/Kusama ecosystem and supports privatize tokens in DOT/SAMA ecosystem via XCM (Cross Chain Messaging). Manta could privatize more tokens via bridges in the future.
```

Comment:
nit: I like 'can' over 'could' (it sounds like we are more likely to bridge other tokens)